analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -74,6 +74,19 @@ object VirtualEnv : CommandLineTool {
     override fun getVersionRequirement(): Requirement = Requirement.buildIvy("15.1.+")
 }
 
+object ProjectVersion : CommandLineTool {
```

Comment:
Commit message nits:
- Please use imperative to describe your changes, see https://github.com/heremaps/oss-review-toolkit/blob/master/CONTRIBUTING.md.
- There is a typo in "detect".
- "Python" should be capitalized.
- So, in your case please use the title "PIP: Add a script to detect the Python version"

Reply:
Why is this object called "ProjectVersion"? Shouldn't it rather be called "PythonVersion"?

Reply:
I think you have right. I will change this name. 

- [x] Change project name.
- [ ] Change commit message.

Reply:
- [x] Squash all commits.